cookbook_se/Dockerfile

```diff
@@ -30,7 +30,7 @@ COPY ./requirements.txt /root
 RUN ${VENV}/bin/pip install -r /root/requirements.txt
 
 # Leaving this in here for easier iteration
-RUN ${VENV}/bin/pip install -U https://github.com/lyft/flytekit/archive/32639010ba72c35aa18a77da31a28ac8c8fea896.zip#egg=flytekit[sidecar,schema]
+RUN ${VENV}/bin/pip install -U https://github.com/lyft/flytekit/archive/65fdd35b4788fe38c3c0796e5df991cacc8ec125.zip#egg=flytekit[sidecar,schema]
```

Comment:
I don't see this commit... can we use one on the annotations branch?

Reply:
oh sure, i was waiting for my fast-register fixes to go in